### PR TITLE
Refs Task#396714 update mapping management

### PR DESCRIPTION
**Pull request by quanghung97**
## Related Tickets

- [#396714](https://dev.sun-asterisk.com/issues/396714)

## What's this PR do ?

- [x] coding update mapping management
- [x] unit test

## Performance

- [ ] Resolved n + 1 query
- [ ] Run explain query already
- [ ] Time run rake task : 1000 ms

## Checklist

- [x] It was tested in local success?
- [x] Fill link PR into ticket and the opposite
- [x] Note purpose/reason, solution, scope of influence into ticket

## ENV note

```
```

## Notes
image

### CODE REVIEW

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr></table>

**Review by ghost — APPROVED** [review]

**Review by tamtt-1031 — APPROVED** [review]

**Review by nguyenthinhhn — APPROVED** [review]

File: app/Policies/MappingPolicy.php [hunk 1]
```diff
@@ -35,4 +35,16 @@ public function create(User $user, Mapping $mapping, Webhook $webhook)
...
+    /**
+     * Determine if the given mapping can be update by user and webhook
+     *
+     * @param App\Models\User
+     * @param App\Models\Webhook
+     * @return bool
+     */
+    public function update(User $user, Mapping $mapping, Webhook $webhook)
+    {
+        return $user->id === $webhook->user_id;
```
Comment (ghost) [inline app/Policies/MappingPolicy.php]: thiếu một trường hợp mapping thuộc webhook khác nè cậu

File: resources/views/mappings/edit.blade.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
+@extends('layouts.app')
+@section('content')
+@include('common.flash-message')
+<link rel="stylesheet" href="{{ mix('/css/style.css') }}">
+
+<ul class="breadcrumb breadcrumb-top">
```
Comment (ghost) [inline resources/views/mappings/edit.blade.php:6]: update lại format của breadcrumb gioongs như [này](https://github.com/sun-asterisk-research/chatwork-forwarder/pull/87/files#diff-1baca90dc66de20916ce7722e5149fd8R6) cho thống nhất giúp mình với nhé :D

**Review by ghost — APPROVED** [review]

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr></table>

**Comment by phucqd-1305** [comment]
OK

**Pull request merged**